### PR TITLE
add triage-new-issues app

### PR DESCRIPTION
**Pull request by olstenlarck**
Adds `triage` label to newly-created issues and then removes it when other label(s) are added.

https://github.com/tunnckoCore/triage-new-issues

Screenshots: https://imgur.com/a/fLX9Z

Notes: i'll add tests soon, currently is one fake test, that is required because of CI and automatic releasing.

### CODE REVIEW

**Comment by olstenlarck** [comment]
@bkeepers any help?

File: _apps/triage-new-issues.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+host: https://triage-new-issues.now.sh
+---
+
+
+## Use programmatically
+
+Install it using [**yarn**](https://yarnpkg.com) **v1** or above / [**npm**](https://www.npmjs.com) **v5** or above.
+
+```
+$ yarn add {%= name %}
```
Comment (j-f1) [inline _apps/triage-new-issues.md]: Looks like you need to fill in the name of the package on npm or remove this section of the readme — `{%= name %}` isn’t valid Liquid.

Comment (olstenlarck) [reply _apps/triage-new-issues.md]: ah, yea. Thanks.

**Comment by bkeepers** [comment]
Thanks, looks great!

**Comment by olstenlarck** [comment]
Thanks too! :tada: 

We should update some of the links too and probably make sense to remove the screenshots that are on the readme.

And one more thing: Does listing here mean that it is now hosted on your side? So i can remove it from my Now account?

**Comment by bkeepers** [comment]
> And one more thing: Does listing here mean that it is now hosted on your side? So i can remove it from my Now account?

No, not necessarily. A lot of these apps are hosted by their original authors. If they become more popular and are too challenging for someone to host on their own, we'll consider taking them on though.

**Comment by olstenlarck** [comment]
Ookey, right. It's not a problem to me currently. Happy using to who's using it :)

**Pull request merged**